### ISSUE TITLE
ADAM upload fails (2sxc v13LTS)

### ISSUE THREAD

**Issue by skarpik** [2sic/2sxc#2828]
**I'm submitting a ...**   
[x] bug report => search github for a similar issue before submitting 

**...about**   
[x] edit experience / UI
[x] admin experience UI

**Current behavior**   
I wrote a 2sxc app using 2sxc v.13.04.00 (Content v13.1.2). It works perfectly.

When I finished the custom app, I created a new website for production with 2sxc v.13.12.01 LTS (Content v13.10.3). Everything worked perfectly except my app now refused to upload images through ADAM. I was suspicious that the problem was not with my code but rather with the underlying 2sxc code.

To investigate, I tried uploading an image into the **Data** for the **Content** app.  As expected, for **2sxc v.13.04.00 (Content v13.1.2)**, the ADAM upload works perfectly. However, for **2sxc v.13.12.01 LTS (Content v13.10.3)**, ADAM refuses to accept the image.

Just for curiosity, I tested the ADAM image upload for 2sxc v14.4.0 (Content v14.7.0) and it works perfectly.

**Expected behavior**   
ADAM uploads should work for the 2sxc v13 LTS and the associated Content app v13.10.3.

**Instructions to Reproduce the Problem**

1. Created test DNN v9.10.2 website. Installed 2sxc 13.12.01 LTS plus all associated Content and Apps.
2. Opened the Data control panel for the Content app
![Screenshot 2022-08-06 124920](https://user-images.githubusercontent.com/3846223/183258557-9f5e13f9-cc2b-42f3-ae9d-6ee7a4412ba7.jpg)
3. Clicked on the "Image" link on the Content Admin page
4. Clicked on the "+" to add new image.
5. Dragged an image to the ADAM interface
![Screenshot 2022-08-06 121451](https://user-images.githubusercontent.com/3846223/183258560-30179c39-f492-4839-a790-9cc198c04b27.jpg)

The dragged image shows up as a flash in the ADAM thumbnail and then vanishes. No upload performed.

**Why change the behavior?**  

ADAM uploads are core functionality. My custom app must be sharing some code that is broken in the ADAM system.

**Your environment**  

* **2sxc version(s):** 13.12.01 LTS (works OK in v14.4.0) 
* **Browser:** Edge, Chrome
* **DNN:** 9.10.2
* **Language:** English

**Anything you would like to add**
The tests that I did were done by an account with Super User privileges.

**Comment by tvatavuk** [comment]
Hi @skarpik, thx for reporting the issue.

I did try to reproduce your issue in DNN 9.10.2 + older 2sxc 13.12.01 LTS (and current 14.07.04 LTS).
I followed your steps and did not have an issue while uploading my usual test images (this is expected to work on default dnn).
After that, I was trying to upload an image that is too large (file size bigger than dnn allows) and then I experienced the behaviour You explained.

This is known issue with file upload. It happens when file upload results with exception (eg. because uploaded file size is larger than allowed in dnn configuration, file extension in not permitted in dnn, etc).

Issue is that file upload error message is not propagated to UI, so user do not see that error happens.

Until we fix this UI issue, advanced user can check in developer tools, on network tab, what is the response of the file upload. Here is my example.

![image](https://user-images.githubusercontent.com/2724850/183367253-3e62fd0f-f34e-4560-b2b8-044765ad4b6d.png).
